packages/tpetra/core/test/CrsMatrix/CMakeLists.txt

```diff
@@ -47,6 +47,16 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   STANDARD_PASS_OUTPUT
   )
 
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  CrsMatrix_UnitTests_Swap
+  SOURCES
+    CrsMatrix_UnitTests_Swap
+    ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  ARGS ${ARGS}
+  COMM serial mpi
```

Comment:
If you want this problem to run only with 2 ranks you should specify it here, also if you want to avoid the case with a single rank in the communicator then specifying `COMM serial` seems wrong...

Reply:
@lucbv 
Ah, ok. I've modified the CMakeLists.txt here to just do mpi and to set it to 2 processes since that's what the test is set up for for now. In the future if we can make the test work for 1..N processes then we can update the CMakeLists.txt accordingly.
Running a quick local build before pushing this change up.